src/lib/components/ui/Search.svelte

```
@@ -0,0 +1,199 @@
+				const response = await client.searchSingleIndex({
+					indexName: 'Article',
+					searchParams: { query }
+				});
+
+				// console.log(response.hits);
+
+				results.set(response.hits as SearchResult[]);
+			} catch (error) {
+				console.error('Algolia search error:', error);
+				results.set([]);
+				const errorMessage = error instanceof Error ? error.message : 'Search failed';
```

ESLint

[error] 'errorMessage' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)